test/main/dump.js

```diff
@@ -1,3 +1,5 @@
+/* globals Symbol:false */
+
```

Comment:
Since Symbol still isn't supported by all browsers a we support, I think it's beneficial to still call it out specifically.